config/sac_trainer_config.yaml

```diff
@@ -0,0 +1,271 @@
+default:
+    trainer: sac
+    batch_size: 128
+    buffer_size: 50000
+    buffer_init_steps: 0
+    hidden_units: 128
+    init_entcoef: 1.0
+    learning_rate: 3.0e-4
+    max_steps: 5.0e4
+    memory_size: 256
+    normalize: false
+    num_update: 1
+    train_interval: 1
+    num_layers: 2
+    time_horizon: 64
+    sequence_length: 64
+    summary_freq: 1000
+    tau: 0.005
+    use_recurrent: false
+    vis_encode_type: default
+    reward_signals:
+        extrinsic:
+            strength: 1.0
+            gamma: 0.99
+
+BananaLearning:
+    normalize: false
+    batch_size: 256
+    buffer_size: 500000
+    max_steps: 1.0e5
+    init_entcoef: 0.05
+    train_interval: 1
+
+VisualBananaLearning:
+    beta: 1.0e-2
+    gamma: 0.99
+    num_epoch: 1
+    max_steps: 5.0e5
+    summary_freq: 1000
+
+BouncerLearning:
+    normalize: true
+    beta: 0.0
+    max_steps: 5.0e5
+    num_layers: 2
+    hidden_units: 64
+    summary_freq: 1000
+
+PushBlockLearning:
+    max_steps: 5.0e4
+    init_entcoef: 0.05
+    beta: 1.0e-2
+    hidden_units: 256
+    summary_freq: 2000
+    time_horizon: 64
+    num_layers: 2
+
+SmallWallJumpLearning:
+    max_steps: 1.0e6
+    beta: 5.0e-3
+    hidden_units: 256
+    summary_freq: 2000
+    time_horizon: 128
+    init_entcoef: 0.1
+    num_layers: 2
+    normalize: false
+
+BigWallJumpLearning:
+    max_steps: 1.0e6
+    beta: 5.0e-3
+    hidden_units: 256
+    summary_freq: 2000
+    time_horizon: 128
+    num_layers: 2
+    init_entcoef: 0.1
+    normalize: false
+
+StrikerLearning:
+    max_steps: 5.0e5
+    learning_rate: 1e-3
+    beta: 1.0e-2
+    hidden_units: 256
+    summary_freq: 2000
+    time_horizon: 128
+    init_entcoef: 0.1
+    num_layers: 2
+    normalize: false
+
+GoalieLearning:
+    max_steps: 5.0e5
+    learning_rate: 1e-3
+    beta: 1.0e-2
+    hidden_units: 256
+    summary_freq: 2000
+    time_horizon: 128
+    init_entcoef: 0.1
+    num_layers: 2
+    normalize: false
+
+PyramidsLearning:
+    summary_freq: 2000
+    time_horizon: 128
+    batch_size: 128
+    buffer_init_steps: 10000
+    buffer_size: 500000
+    hidden_units: 512
+    num_layers: 2
+    init_entcoef: 0.5
+    max_steps: 5.0e5
+    sequence_length: 16
+    use_recurrent: false
+    reward_signals:
+        extrinsic:
+            strength: 1.0
+            gamma: 0.99
+        curiosity:
```

Comment:
I actually think it will learn once we implement n-step Bellman backup. It's having trouble bootstrapping on such a sparse reward, but the curiosity module does eventually lead it to reach the goal occasionally. 

Just tried GAIL - it's learning. So I replaced it with that for now. 

We actually already committed all the demos for all of the example environments in the previous release, conveniently.